src/routes.js

```diff
@@ -82,6 +82,55 @@ export default [
     name: `block`,
     component: require(`./components/network/PageBlock`).default
   },
+  {
```

Comment:
Can we tell Code Climate to ignore this file?

<img width="734" alt="Screenshot 2019-07-03 at 15 08 40" src="https://user-images.githubusercontent.com/360865/60594091-7a167400-9da4-11e9-9392-17fdbec60d06.png">

Reply:
It's not a required check

Reply:
Sure, but I think it's a good habit to treat warnings and errors as much the same. In this case there is nothing wrong with this code.

Reply:
I marked the errors at `wontfix`